server/src/services/formatting/onDocumentFormatting.ts

```diff
@@ -33,7 +33,7 @@ export function onDocumentFormatting(serverState: ServerState) {
       }
     } catch (error) {
```

Comment:
Can we differentiate between errors that are due to syntax problems and ignore them, and pass on other errors to sentry as an error.

Reply:
What use would we have for them ? possible cases would be that `forge` could not be resolved (not installed), or some config errors. This would be shown to the user on the logs. I thought sentry errors would be for unhandled situations (maybe I'm wrong in this understanding)